src/geo/ui/legends/bubble/legend-template.tpl

```diff
@@ -1,18 +1,32 @@
 <div class="Bubble-container u-flex u-justifySpace">
   <ul class="Bubble-numbers u-flex u-justifySpace">
-    <% for (var i = 0; i<labels.length; i++) { %>
-      <li class="Bubble-numbersItem CDB-Text CDB-Size-small" style="bottom: <%- labelPositions[i] %>%"><%= formatter.formatNumber(labels[i]) %></li>
+    <% if (hasCustomLabels) { %>
+      <li class="Bubble-numbersItem CDB-Text CDB-Size-small" style="bottom: 0%"><%- labels[0] %></li>
```

Comment:
That was my first idea but the thing is the custom labels come from the user, so we need to avoid `<%= whatever %>` in order to prevent code injection. For data coming from inside, we need html to be interpreted in some cases. Not sure how to solve this.